apptools/logger/logger.py

```diff
@@ -29,10 +29,24 @@ class LogFileHandler(RotatingFileHandler):
     """The default log file handler."""
 
     def __init__(
-        self, path, maxBytes=1000000, backupCount=3, level=None, formatter=None
+        self,
+        path,
+        mode="a",
+        maxBytes=1000000,
+        backupCount=3,
+        level=None,
+        formatter=None,
+        encoding=None,
+        delay=False,
     ):
         RotatingFileHandler.__init__(
-            self, path, maxBytes=maxBytes, backupCount=3
+            self,
+            path=path,
```

Comment:
```suggestion
            filename=path,
```

It looks like the kwarg is called `filename` on the `RotatingFileHandler`, not `path` like what we're using - https://github.com/python/cpython/blob/8d999cbf4adea053be6dbb612b9844635c4dfb8e/Lib/logging/handlers.py#L122